x-pack/test/plugin_api_integration/plugins/sample_task_plugin/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "sample_task_plugin",
+  "version": "1.0.0",
+  "kibana": {
+    "version": "kibana",
+    "templateVersion": "1.0.0"
+  },
+  "main": "target/test/plugin_api_integration/plugins/sample_task_plugin",
+  "scripts": {
+    "kbn": "node ../../../../../scripts/kbn.js",
+    "build": "rm -rf './target' && tsc"
+  },
+  "devDependencies": {
+    "typescript": "3.7.2"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "joi": "^13.5.2"
```

Comment:
Do we not get joi "for free", as in from `/x-pack/node_modules` or `/node_module`?  Same with `typescript` above.   My only real worry is that with the version specificity there, could be interesting to debug when the base versions get upgraded, but this plugin is using older versions.

Reply:
Good catch actually as we don't need Joi anymore. :)
Sadly the TS has to be specified.. I learned this the hard way a few months ago when a fixture consistently failed on CI for this reason.